src/main/resources/static/views/user/UserController.js

```diff
@@ -37,24 +36,14 @@ var UserController = angular.module('greatStartApp')
         };
 
         $scope.submit = function () {
-            if ($scope.user.id === null) {
-                console.log('Saving New User', $scope.user);
-                $scope.createUser();
-            } else {
-                if ($scope.isImageChange) {
-                    $scope.user.photo = $scope.myCroppedImage.replace(/^data:image\/[a-z]+;base64,/, "");
-                }
-                $scope.update($scope.user).$promise.then(function (success) {
-                    $location.path('/user/:id', $scope.user.id);
-                },function (error) {
-                    $scope.error=true;
-                });
-
+            if ($scope.isImageChange) {
+                $scope.user.photo = $scope.myCroppedImage.replace(/^data:image\/[a-z]+;base64,/, "");
             }
-        };
-
-        //TODO write create method for user
-        $scope.createUser=function () {
+            $scope.update($scope.user).$promise.then(function (success) {
```

Comment:
I didn't change it since I've created it in my #76 task. And without promise it doesn't work